_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,294 @@
+---
+name: Iframes with identical accessible names serve equivalent purpose
+
+description: |
+ This rule checks that iframes with identical accessible names serve an equivalent purpose
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to `iframe` elements that are [included in the accessibility tree](#included-in-the-accessibility-tree) and have the same [non-empty](#non-empty) [accessible name](#accessible-name).
```

Comment:
The term "the same" is not well-defined.

Reply:
Maybe use "matching-characters" from this PR: https://github.com/auto-wcag/auto-wcag/pull/452

Reply:
Note to self:

..and have [matching](matching-characters) [accessible name](#accessible-name), that do not only include whitespace.

and see pull request #430